### PR TITLE
Signup: Headstart: Add back the Headstart flow to EN-only signups.

### PR DESCRIPTION
**Pull request by kwight**

### CODE REVIEW

**Comment by kwight** [comment]
We'll need to check how this works with English language variations, eg. EN-GB and EN-CA.

**Comment by kwight** [comment]
EN-GB is supported, EN-CA falls back to EN. A quick scan of Headstart content doesn't show anything major that would require EN-GB specific translation.

**Comment by kwight** [comment]
@sirbrillig for review

File: client/signup/config/flows.js [hunk 1]
```diff
@@ -228,7 +229,8 @@ function removeUserStepFromFlow( flow ) {
 }
 
 function filterFlowName( flowName ) {
-	if ( 'headstart' === flowName && 'production' === config( 'env' ) ) {
+	const locale = getLocaleSlug();
+	if ( 'headstart' === flowName && 'en' !== locale && 'en-gb' !== locale ) {
 		return 'main';
```
Comment (sirbrillig) [inline client/signup/config/flows.js]: Maybe you wanted to return 'headstart' here?

Comment (kwight) [inline client/signup/config/flows.js]: No – that's the check that says, "if we're in the Headstart flow, but the user's locale isn't EN, send them to `main` instead" (which is localized).

Comment (sirbrillig) [inline client/signup/config/flows.js]: Ah! got it. Cool.

**Comment by sirbrillig** [comment]
LGTM.

**Pull request merged**